.circleci/create_circleci_config.py

```diff
@@ -554,6 +567,17 @@ def create_circleci_config(folder=None):
     if len(test_list) > 0:
         jobs.extend(REGULAR_TESTS)
 
+        test_list_items = test_list.split()
+        if "tests/models" in test_list_items:
+            test_list_items = [x for x in test_list_items if x != "tests/models"]
+            test_list_items += sorted([x for x in glob.glob("tests/models/*/**.py") if "test_" in x])
+
+        for job in jobs:
+            if job.job_name in ["tests_torch", "tests_tf", "tests_flax"]:
+                job.tests_to_run = [x for x in test_list_items if "/test_modeling_" in x]
+            elif job.job_name == "tests_non_modeling":
+                job.tests_to_run = [x for x in test_list_items if "/test_modeling_" not in x]
```

Comment:
I know you don't like these operations. More than happy to save those post-processed lists to files for reference. Or even produce these in test fetcher directly.